vllm/core/scheduler.py

```diff
@@ -152,7 +152,8 @@ def is_empty(self) -> bool:
     def _sort_by_lora_ids(self):
         self.scheduled_seq_groups = sorted(
             self.scheduled_seq_groups,
-            key=lambda g: (g.seq_group.lora_int_id, g.seq_group.request_id))
+            key=lambda g: (not g.seq_group.is_prefill(), g.seq_group.
```

Comment:
can you tell me the motivation of this change?

Reply:
when chunked prefill is disabled, i guess `g.seq_group.is_prefill()` will all be the same value, so it wouldn't change the ordering?

Reply:
ah I see. yeah I think that's right